model/project_ref.go

```diff
@@ -2284,53 +2284,59 @@ func GetBuildBaronSettings(projectId string, version string) (evergreen.BuildBar
 	return projectRef.BuildBaronSettings, true
 }
 
-func BbProjectIsValid(projName string, proj evergreen.BuildBaronSettings) error {
-	webHook, _, err := IsWebhookConfigured(projName, "")
-	if err != nil {
-		return err
+func ValidateBbProject(projName string, proj evergreen.BuildBaronSettings, webhook *evergreen.WebHook, lookupProjectRef bool) []string {
```

Comment:
Returning a list of strings as errors is a bit unconventional (ValidateProjectAliases I think does it but probably shouldn't). If all of these count as errors for validation, we can just use a catcher here and return a singular error, like the periodic build validation.